.ci_support/osx64.yaml

```diff
@@ -1,3 +1,5 @@
+CONDA_BUILD_SYSROOT:                                                                                    # [osx]
+  - /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk  # [osx]
```

Comment:
@conda-forge/help-c-cpp - is this right?

Reply:
Nope. This change should be removed.

Reply:
Had to rebase to the latest master since this was an old PR. Should be good to go now.